cloudbuild.yaml

```diff
@@ -60,10 +60,10 @@ steps:
 - id: lint
   name: 'gcr.io/${PROJECT_ID}/trillian_testbase'
   entrypoint: bash
-  args:
+  args: # TODO(pavelkalinnikov): Upgrade the linter automatically.
     - -exc
     - |
-      curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.21.0
+      curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.36.0
```

Comment:
Hmm, presumably if this is installed in the `trillian_testbase` image, we don't need to install it here at all?

Reply:
Right. Removed.